src/js/view/controls/tizen/tizen-controlbar.ts

```diff
@@ -60,16 +64,6 @@ function getNextButton(activeButton: Button, layout: ControlbarElement[], toRigh
     return;
 }
 
-function getButtonElement(btn: any): HTMLElement | undefined {
```

Comment:
Wondering if this was meant to be removed? This had been added so that the adSkipButton could also be passed to `setActiveButton` below. Then `this.activeButton` wouldn't be null when there actually was a button active.

Reply:
I made this branch when the following pattern was implemented for case 39:

![image](https://user-images.githubusercontent.com/2903955/96377662-09d86100-1155-11eb-8b3a-f879cb424ed5.png)

I've just updated it to have parity with the new logic.